core/lib/Thelia/Core/EventListener/FormListener.php

```diff
@@ -0,0 +1,89 @@
+<?php
+/*************************************************************************************/
+/*      This file is part of the Thelia package.                                     */
+/*                                                                                   */
+/*      Copyright (c) OpenStudio                                                     */
+/*      email : dev@thelia.net                                                       */
+/*      web : http://www.thelia.net                                                  */
+/*                                                                                   */
+/*      For the full copyright and license information, please view the LICENSE.txt  */
+/*      file that was distributed with this source code.                             */
+/*************************************************************************************/
+
+namespace Thelia\Core\EventListener;
+
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Thelia\Controller\Admin\CustomerController;
+use Thelia\Core\Controller\ControllerResolver;
+use Thelia\Core\Event\TheliaEvents;
+use Thelia\Core\Event\TheliaFormEvent;
+use Thelia\Core\HttpFoundation\Request;
+
+/**
+ * Class FormListener
+ * @package Thelia\Core\EventListener
+ * @author Manuel Raynaud <manu@thelia.net>
+ */
+class FormListener implements EventSubscriberInterface
+{
+
+    /**
+     * @var Request
+     */
+    protected $request;
+
+    /**
+     * @var ControllerResolver
+     */
+    protected $resolver;
+
+    /**
+     * @param Request $request
+     * @param ControllerResolver $resolver
+     */
+    public function __construct(Request $request, ControllerResolver $resolver)
+    {
+        $this->request = $request;
+        $this->resolver = $resolver;
+    }
+
+    public function removeEmailConfirmation(TheliaFormEvent $event)
+    {
+        $controller = $this->resolver->getController($this->request);
+
+        if ($controller[0] instanceof CustomerController) {
```

Comment:
For the front context you can set the value of ```customer_confirm_email``` to 0 or 1 if you want to verify the email or not